notebooks/community/pipelines/google_cloud_pipeline_components_cloud_natural_language_pipeline.ipynb

```diff
@@ -0,0 +1,813 @@
+{
```

Comment:
<p>per template:</p><p>Missing objective section</p><p>Should be split into separate cells: Overview, Objective, Dataset, Costs</p><p> </p><p>Vertex dataset -&gt; Vertex AI Dataset</p><p>Vertex model registry -&gt; Vertex AI Model Registry</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done.